server_globals.lua

```diff
@@ -30,6 +30,9 @@ PLACEMENT_MATCH_K = 50
 NAME_LENGTH_LIMIT = 16
 PLACEMENT_MATCHES_ENABLED = false
 COMPRESS_REPLAYS_ENABLED = false
+COMPRESS_SPECTATOR_REPLAYS_ENABLED = false -- Send current replay inputs over the internet in a compressed format to spectators who join.
+TCP_NODELAY_ENABLED = false -- Disables Nagle's Algorithm for TCP. Decreases data packet delivery delay, but increases amount of bandwidth and data used.
```

Comment:
shouldn't TCP_NODELAY_ENABLED be enabled by default since its the better option and the whole point of this PR?

Reply:
Yea, I don't remember why I turned it off by default
```suggestion
TCP_NODELAY_ENABLED = true -- Disables Nagle's Algorithm for TCP. Decreases data packet delivery delay, but increases amount of bandwidth and data used.
```